pkg/runner/step_context.go

```diff
@@ -91,6 +91,13 @@ func (sc *StepContext) Executor(ctx context.Context) common.Executor {
 
 		remoteAction.URL = rc.Config.GitHubInstance
 
+		if remoteAction.IsCheckout() && rc.Config.DisableLocalCheckout {
```

Comment:
That's not correct. What I meant, when I talked with you about this was:
https://github.com/nektos/act/blob/b7f0a7de234ef2c3065a41e4a5d0b59d0f128828/pkg/runner/step_context.go#L103 returns a `nil` executor instead of actually executing the `actions/checkout` in act.
So I meant a flag to disable this "skipping" behavior of act, so that when the flag is set, act will always execute `actions/checkout` when it is present in a workflow just like any other action.

Reply:
The default should continue to be that act copies / bind-mounts the workdir into the container, so that it is easy to develop and debug actions and workflows locally.
But in some cases, you might want to actually run the `actions/checkout` action instead of letting act handle the copy/mount.

Reply:
I agree with @ZauberNerd. I have an implementation for forceremotecheckout, but my backports got stuck a while ago https://github.com/ChristopherHX/act/blob/54588b02c3e0cef063635341c0163553fe571dd5/pkg/runner/runner.go#L54

FYI My backport progress is blocked by https://github.com/nektos/act/pull/1004

Reply:
oh i misunderstood @ZauberNerd